Maintenance Announcement EU (#2031)
galaxyproject/galaxy-hub: content/news/2023-06-06-eu-maintenance/index.md

---
title: "EU Maintenance Announcement 12th June"
date: "2023-06-06"
tease: "EU database maintenance, 8hrs downtime estimated"
hide_tease: false
authors: 'Sanjay Srikakulam, José Manuel Domínguez, Mira Kuntz'
authors_structured:
- github: sanjaysrikakulam
- github: kysrpex
- github: mira-miracoli
tags: [EU]
subsites: [all-eu]
main_subsite: eu
---

# Database maintenance on 12 June

Maintenance will take place on 12 June from 9:30am CEST (7:30am GMT) and should not exceed 5:30pm CEST (3:30pm GMT). usegalaxy.eu will not be reachable during this period.

## Updating our database and installing new hardware

> Our database is currently running stable and reliable, but it is running on CentOS 7. Since the CentOS project no longer exists and the repositories are no longer maintained, we need to change the operating system on our database server.  Therefore, we will update the server to Rocky Linux 9.

Reviewer: ```suggestion
Our database server is currently running stable and reliable, but it is running on CentOS 8. Since the CentOS project no longer exists and the repositories are no longer maintained, we need to change the operating system on our database server.  Therefore, we will update the server to Rocky Linux 9.
```

> In order to use this downtime as efficiently as possible, we thought of upgrading the server hardware with two 4TB SSD disks and make it ready for an ever-growing Galaxy community.  

Reviewer: ```suggestion
To use this downtime as efficiently as possible, we thought of upgrading the server hardware with two 4TB SSD disks and making it ready for an ever-growing Galaxy community.  
```

> By now, Galaxy Europe has 70,000 registered users and counting!  

Reviewer: ```suggestion
Currently, Galaxy Europe has 70,000 registered users and counting!  
```
